python/ql/src/experimental/Security/CWE-089/ClickHouseSQLInjection.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @id py/yandex/clickhouse-sql-injection
+ * @name Clickhouse SQL query built from user-controlled sources
+ * @description Building a SQL query from user-controlled sources is vulnerable to insertion of
+ *              malicious SQL code by the user.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @tags security
+ *       external/cwe/cwe-089
+ *       external/owasp/owasp-a1
```

Comment:
Is it possible to reference `owasp-a1` here?

Reply:
Why not? `owasp-a1` is about injections.

Reply:
I was asking because I never saw such an external reference before but a quick `grep` revealed that there are already existing references to `owasp-a1`.

Feel free to resolve this comment.